packages/typescript/src/lib/TypescriptCompiler.ts

```diff
@@ -100,7 +95,7 @@ export class TypescriptCompiler {
 
       return (
         destFile &&
-        path.join(this.compilerOptions.outDir as string, `${destFile}.d.ts`)
+        path.join(this.compilerOptions.outDir as string, 'index.d.ts')
```

Comment:
for multiple components, this line makes one component override another... ⚠️ 
so when before we had "button" and "another-button" generating "button.d.ts" and "another-button.d.ts",
but with this change, both of these components generate "index.d.ts" so the first "button.d.ts" content is lost.

Reply:
This has been addressed